core/src/components/modal/test/card-scroll-target/modal.e2e.ts

```diff
@@ -0,0 +1,64 @@
+import { expect } from '@playwright/test';
+import { dragElementBy, test } from '@utils/test/playwright';
+
+test.describe('card modal - scroll target', () => {
```

Comment:
Is this still needed? Looks like everything is duplicated in the `card` test.

Reply:
This test duplicates the tests in `card` but with the `.ion-content-scroll-host` element. I am trying to check that we are looking at the correct `scrollTop` values when swiping. For example, when using `.ion-content-scroll-host` the scrollTop value of `ion-content`'s inner scroll element will always be `0`. But in that case, we want to check the `.ion-content-scroll-host` scrollTop instead.

Reply:
Ah, gotcha!